load.Rmd

```diff
@@ -641,7 +638,7 @@ if(!"gendereq_slc" %in% scens)
 # Rename columns
 l@data <- rename(l@data, Home = Area.of.residence, Work = Area.of.workplace)
 names(zones@data)
-
+l$clc_perc <- l$clc * 100
```

Comment:
Any reason we need this?  It means we have `clc` and `clc_perc` in the line data and I'm not sure of the benefit